src/views/Editor/index.js

```diff
@@ -118,10 +119,11 @@ var EditorPage = React.createClass({
         } else {
             return (
                 <span>
+                    <Loader loaded={!this.props.editor.isSending} scale={1}/>
                     <RaisedButton
                         style={buttonStyle}
                         primary={true}
-                        disabled={this.props.user && !this.props.user.organization}
+                        disabled={this.props.editor.isSending || (this.props.user && !this.props.user.organization)}
                         label={labelText}
```

Comment:
Trying this out, in some cases it really takes a long while to publish an event. Should we change the button text from labelText ("julkaise tapahtuma") to publishing ("julkaistaan tapahtumaa...") to make the user understand why they have to wait and the button is disabled?